app/src/main/res/values/strings.xml

```diff
@@ -116,17 +116,17 @@
 
     <string name="enter_group_name">Enter group name</string>
     <string name="groups">Groups</string>
-    <string name="noGroups">Click the "+" (plus) button to add groups first.\n\nGroups make things easier to find.</string>
+    <string name="noGroups">Click the "+" (plus) button to add groups for categorization first.</string>
     <plurals name="groupCardCount">
         <item quantity="one"><xliff:g>%d</xliff:g> card</item>
         <item quantity="other"><xliff:g>%d</xliff:g> cards</item>
     </plurals>
 
     <string name="all">All</string>
-    <string name="deleteConfirmationGroup">Please confirm you want to delete this group</string>
+    <string name="deleteConfirmationGroup">Delete group?</string>
     <string name="failedOpeningFileManager">Install a file manager first.</string>
-    <string name="moveUp">Move up in list</string>
-    <string name="moveDown">Move down in list</string>
+    <string name="moveUp">🡅</string>
+    <string name="moveDown">🡇</string>
```

Comment:
Please do not change these to arrows. The arrows are images, these are the descriptions for people using accessibility tools.